cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/ExecutorConfig.java

```diff
@@ -56,7 +58,8 @@ public static ExecutorConfig create(
             @JsonProperty("executor_location") String executorLocation,
             @JsonProperty("cassandra_location") String cassandraLocation,
             @JsonProperty("libmesos_location") String libmesosLocation,
-            @JsonProperty("emc_ecs_workaround") boolean emcEcsWorkaround)
+            @JsonProperty("emc_ecs_workaround") boolean emcEcsWorkaround,
+            @JsonProperty("cache_fetched_uris") boolean cacheFetchedUris)
```

Comment:
We fetch the URIs and then cache the fetched URIs. I am open to suggestions for better names.